diff --git a/extras/Makefile.getdeps b/extras/Makefile.getdeps
index 9df3ffdd7..0bf55af54 100644
--- a/extras/Makefile.getdeps
+++ b/extras/Makefile.getdeps
@@ -22,7 +22,7 @@ MPC_VERSION=1.0.3
 MPFR_VERSION=3.1.3
 GMP_VERSION=6.1.0
 CONTRIB_OPT=-O3
-NUMACTL_VERSION=2.0.8
+NUMACTL_VERSION=2.0.12
 
 EXAMPLE_DEPS=$(CONTRIB_LIBDIR)/libblas.so \
 	$(CONTRIB_LIBDIR)/liblapack.so \
@@ -485,7 +485,7 @@ $(CONTRIB_DIR)/numactl-$(NUMACTL_VERSION)/Makefile: $(ARCHIVES_DIR)/numactl-$(NU
 	if [ -d $(CONTRIB_DIR)/numactl-$(NUMACTL_VERSION) ] ; \
 		then rm -rf "$(CONTRIB_DIR)/numactl-$(NUMACTL_VERSION)" ; fi
 	cd $(CONTRIB_DIR); tar xf $<
-	touch $@
+	cd $(CONTRIB_DIR)/numactl-$(NUMACTL_VERSION); ./configure --prefix=$(INST_DIR)
 
 $(ARCHIVES_DIR)/numactl-$(NUMACTL_VERSION).tar.gz:
 	mkdir -p $(ARCHIVES_DIR)
